WordPress/Sniffs/NamingConventions/PrefixAllGlobalsSniff.php

```diff
@@ -58,13 +58,13 @@ final class PrefixAllGlobalsSniff extends AbstractFunctionParameterSniff {
 	/**
 	 * Minimal number of characters the prefix needs in order to be valid.
 	 *
-	 * @since 2.2.0
+	 * @since n.e.x.t.
 	 *
-	 * @link https://github.com/WordPress/WordPress-Coding-Standards/issues/1733 Issue 1733.
+	 * @link https://github.com/WordPress/WordPress-Coding-Standards/issues/2467 Issue 2467.
```

Comment:
Same here, I'd add this as a separate `@link`.

Reply:
ok!